roles/operator_bundle_scorecard_tests/files/optional-operators-cvp-common-scorecard-commands.sh

```diff
@@ -3,32 +3,37 @@
 set -o nounset
 set -o errexit
 set -o pipefail
+set -x
```

Comment:
Why do we need trace mode in the script ?

Reply:
I don't think this could ever hurt. Bash scripts are hard to debug as it is, and having more debug information if something goes wrong could be helpful.

Reply:
Bill had requested that `sed -x` be added so the script was easier to debug.  I don't think there's anything wrong with leaving it there.  If you disagree I can change it, but I'll have to submit a PR to openshift/release as well

Reply:
If somebody want do debug script they can  run `bash -x $name of script` without changing the content.

That file is executed from ansible playbook and it is difficult to read very verbose tracing log in the ansible output unless you run playbook with `-vvv`